bibblio.py

```diff
@@ -237,32 +249,21 @@ def process_item(self, identifier, force=False):
             self._db, Identifier.BIBBLIO_CONTENT_ITEM_ID, content_item_id
         )
 
-        identifier.equivalent_to(
-            self.output_source, bibblio_identifier, 1
-        )
-        return identifier
-
-    def add_coverage_record_for(self, item):
-        """Adds an appropriate CoverageRecord for this particular
-        identifier and any equivalent identifiers.
-
-        Because Bibblio is text-based recommendation source, it doesn't
-        make sense to represent the same work multiple times.
-        """
-        equivalent_identifier_ids = item.equivalent_identifier_ids()[item.id]
-
-        equivalent_identifiers = self._db.query(Identifier).filter(
-            Identifier.id.in_(equivalent_identifier_ids)
+        work_identifiers = self._db.query(Identifier).filter(
+            Identifier.id.in_(work.all_identifier_ids())
         )
-        for identifier in equivalent_identifiers:
-            super(BibblioCoverageProvider, self).add_coverage_record_for(identifier)
+        for identifier in work_identifiers:
+            identifier.equivalent_to(
+                self.output_source, bibblio_identifier, 1
```

Comment:
That's good for now. In the future maybe a connection between Work and Identifier makes sense.